apparmor.d/groups/bus/dbus-daemon

```diff
@@ -47,6 +48,7 @@ profile dbus-daemon @{exec_path} flags=(attach_disconnected) {
   /{usr/,}lib/@{multiarch}/tumbler-1/tumblerd    rPUx,
 
   /usr/share/org.gnome.Characters/org.gnome.Characters.BackgroundService rPx,
+  /usr/share/gnome-documents/org.gnome.Documents rPx,
```

Comment:
Where is the profile for this?

Reply:
Nowhere yet. I felt reluctant setting `rPUx` for minor processes from major ones (`dbus-daemon`, `pkexec`). There no fatal errors. You still want `rPUx`?

Reply:
Yes, otherwise this is a bug at run time.